src/Agriculture.jl

```diff
@@ -140,3 +140,11 @@ function grad_agriculture_allcropproduction_irrcropproduction(m::Model)
     end
     roomintersect(m, :Agriculture, :allcropproduction, :irrcropproduction, gen)
 end
+
+
+function constraintoffset_agriculture_sorghumarea(m::Model)
```

Comment:
Please name this to be Colorado-specific.

Reply:
done